guides/v2.2/pattern-library/general/accessibilityguideline/accessibilityGuideline.md

```diff
@@ -6,11 +6,11 @@ The Magento application should be accessible for all people, whatever their hard
 
 ## Visual
 
-#### Be mindful of text length
```

Comment:
This doesn't seem like a good use of h3.

Reply:
Yeah, I agree... What do you suggest @dshevtsov? Maybe we can convert the H3s to unordered lists?

Reply:
See 1282a6b